iceoryx_posh/source/roudi/roudi_cmd_line_parser.cpp

```diff
@@ -142,6 +147,21 @@ void CmdLineParser::parse(int argc, char* argv[], const CmdLineArgumentParsingMo
             }
             break;
         }
+        case 'f':
+        {
+            uint32_t finalKillTimeInSeconds{0u};
+            constexpr uint64_t MAX_FINAL_KILL_TIME = ((1ul << 32) - 1);
```

Comment:
`std::numeric_limits<uint64_t>::max`?

Reply:
should actually be std::numeric_limits<uint32_t>::max

Reply:
Done.